extensions/policy/ids-policy/src/test/java/org/eclipse/dataspaceconnector/ids/policy/MockPolicyContext.java

```diff
@@ -0,0 +1,50 @@
+/*
+ *  Copyright (c) 2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+package org.eclipse.dataspaceconnector.ids.policy;
+
+import org.eclipse.dataspaceconnector.spi.contract.agent.ParticipantAgent;
+import org.eclipse.dataspaceconnector.spi.contract.policy.PolicyContext;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.ArrayList;
+import java.util.List;
+
+class MockPolicyContext implements PolicyContext {
```

Comment:
It's equal to `PolicyContextImpl`

Reply:
OK, let's remove that in a subsequent PR